### DISCUSSION TITLE
Can You add password to Access Index

### DISCUSSION THREAD

**Discussion by Irupc** [developeranaz/Mega.nz-Rclone-Index-Heroku#10] [Ideas]
if you can add Password for This Mega Index. Its Very Useful for many of Us. 
Because Many People Use Google Drive Index with Passwords.

**Comment by developeranaz** [comment]
> if you can add Password for This Mega Index. Its Very Useful for many of Us.
> Because Many People Use Google Drive Index with Passwords.

It's possible, due to lack of time and some limitations I have ignored it. I will add this feature soon
